src/sidebar/templates/annotation-header.html

```diff
@@ -23,12 +23,12 @@
     <br>
     <span class="annotation-header__share-info">
       <a class="annotation-header__group"
-        target="_blank" ng-if="vm.group() && vm.group().url" href="{{vm.group().url}}">
+        target="_blank" ng-if="vm.group() && vm.group().links.html" href="{{vm.group().links.html}}">
         <i class="h-icon-group"></i><span class="annotation-header__group-name">{{vm.group().name}}</span>
       </a>
       <span ng-show="vm.isPrivate"
         title="This annotation is visible only to you.">
-        <i class="h-icon-lock"></i><span class="annotation-header__group-name" ng-show="!vm.group().url">Only me</span>
+        <i class="h-icon-lock"></i><span class="annotation-header__group-name" ng-show="!vm.group().links.html">Only me</span>
```

Comment:
Can you help me understand the logic here? It appears from first glance that the `span.annotation-header__group-name` is "shown" (`ng-show` will unhide the element via CSS) only if there is no link present for the group's activity page. What is `!vm.group().links.html` actually testing for, conceptually?

Reply:
So, if I remember right, this was a design decision made for publisher groups - to show the 'only me' text.
![image](https://user-images.githubusercontent.com/502186/38836996-51aae7fc-419e-11e8-8200-459e876a9283.png)
![image](https://user-images.githubusercontent.com/502186/38837034-75cb6a76-419e-11e8-82de-5cf4bfb3c20c.png)

Reply:
@ajpeddakotla just confirmed that >> it only appears on 3rd party sites, but I think that this was an eLife requested language change